collector/mdadm_linux.go

```diff
@@ -138,12 +163,20 @@ func parseMdstat(mdStatusFilePath string) ([]mdStatus, error) {
 		}
 		currentMD = mainLine[0]               // name of md-device
 		isActive := (mainLine[2] == "active") // activity status of said md-device
+		personality = mainLine[3]             // Personality type of md-device.
 
 		if len(lines) <= i+3 {
```

Comment:
Yes, remember that we're looking from the context of the mainLine.  So the next 3 lines would be the size info line, a blank line, and the unused devices line.